steps/apptestai-test/0.0.3/step.yml

```diff
@@ -0,0 +1,53 @@
+title: apptestai-test
```

Comment:
pls update based on https://devcenter.bitrise.io/contributors/create-your-own-step/#the-title